_spinnaker/using_dinghy.md

```diff
@@ -468,6 +468,65 @@ If you have already created a pipeline in the Spinnaker UI, you can create a din
    ]
 }
 ```
+4. Add the following parameters to each pipeline within the root of its json
```

Comment:
Can we make this "Add the following parameters to each pipeline **in the collection** ...".

Reply:
Added